tac/project-updates/2025/2025-annual-lockness.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: 2025 Annual Review Lockness
+parent: 2025
+grand_parent: Project Updates
+---
+
+## Project Health
+
+Lockness is not yet onboarded to LFX Insights.
+
+## Maintainer Diversity
+
+We currently have two maintainers: [@survived] and [@maurges] from Dfns. Maintainers did not change since the last year.
+
+Maintainers file per projects:
+* CGGMP21: https://github.com/LFDT-Lockness/cggmp21/blob/m/MAINTAINERS.md
+* givre: https://github.com/LFDT-Lockness/givre/blob/m/MAINTAINERS.md
+* round-based: https://github.com/LFDT-Lockness/round-based/blob/m/MAINTAINERS.md
+* etc.
+
+[@survived]: https://github.com/survived
+[@maurges]: https://github.com/maurges
+
+## Project Adoption
+
+We see that several companies that are trying out our project: they reach out to us asking questions about the project. We don't know for sure if any company is using the project in production (except ourselves).
+
+## Goals
+
+### Performance Against Prior Goals
+
+In [project proposal](https://github.com/dfns/hyperledger-hip-private/blob/proposal/HIPs/incubation/xkey.md), we set and accomplished following goals:
```

Comment:
This link doesn't work. Perhaps you can use https://github.com/LF-Decentralized-Trust/project-proposals/blob/gh-pages/proposals/incubation/xkey.md instead.